nengo/simulator.py

```diff
@@ -58,6 +58,12 @@ def __len__(self):
 class Simulator(object):
     """Reference simulator for Nengo models."""
 
+    unsupported = [('test_constant_scalar', 'who cares')]
```

Comment:
Is this supposed to have content in it? Or is that just an example that we should remove/put in the docstring below?